### PR TITLE
Run hsts-prune and fix impacted rulesets

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

**Comment by Hainish** [comment]
Closing temporarily to fix https://github.com/EFForg/https-everywhere/issues/15213

File: src/chrome/content/rules/Financial_Times.xml [hunk 1]
```diff
@@ -40,7 +40,6 @@
 -->
 <ruleset name="Financial Times (partial)">
 	<target host="ft.com" />
-	<target host="www.ft.com" />
```
Comment (Giltyhub) [inline src/chrome/content/rules/Financial_Times.xml]: One question: Why only www.ft.com is removed when ft.com itself is preloaded?

File: src/chrome/content/rules/Python.org.xml [hunk 2]
```diff
@@ -56,7 +56,6 @@
 	<target host="packaging.python.org" />
 	<target host="packages.python.org" />
 	<target host="pl.python.org" />
-	<target host="pypi.python.org" />
```
Comment (Giltyhub) [inline src/chrome/content/rules/Python.org.xml]: same question for here and a couple of other places

File: src/chrome/content/rules/Financial_Times.xml (hunk 1 above)
Comment (Bisaloo) [reply src/chrome/content/rules/Financial_Times.xml]: It's missing the `includeSubDomains` directive.

See https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#hsts-preloaded-rules and https://securityheaders.com/?q=https%3A%2F%2Fft.com%2F

**Comment by Hainish** [comment]
Thank you!

**Pull request merged**